recipes/combo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "combo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3c5da2b8af17e03146ce382d10150373a260ce64565963f7c0b0cecfc7d093be
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
```suggestion
    - python
    - setuptools >=38.6.0
```

Reply:
people should not add setuptools if they have pip